Doc/library/inspect.rst

```diff
@@ -1439,8 +1439,8 @@ code execution::
            pass
 
 
-Current State of Generators and Coroutines
-------------------------------------------
+Current State of Generators, Coroutines, and Asynchronous Generators
```

Comment:
You also need to add a What's New Entry in `Doc\whatsnew\3.12.rst` under inspect module.

Reply:
Added a new entry